src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -139,6 +139,27 @@ def json2vals(in_x, vals, out_y, name=ROOT):
         for i, temp in enumerate(in_x):
             json2vals(temp, vals[i], out_y, name + '[%d]' % i)
 
+def nni_params2tuner_params(in_x, parameter):
```

Comment:
the name of this function is a little weird? why we need the prefix "nni"?

Reply:
I also confuse the name of this function. This function converts the form of hyperparameters in NNI into the form in TPE tuner. Do you have any suggestions for the name of this function?

Reply:
Rename to "params2tuner_params"